src/components/sharing/ShareBox/style.scss

```diff
@@ -0,0 +1,97 @@
+$shareBoxWidth: 320px;
+
+.ShareBox {
+  box-sizing: border-box;
+  width: $shareBoxWidth;
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  margin-left: -($shareBoxWidth/2);
+  margin-top: $x-2 + 1;
+  background: #fff;
+  border-radius: $ehw-border-radius;
+  padding: $x-4;
+  border: 1px solid $ehw-grey-light;
+  box-shadow: 0 2px 3px rgba(0,0,0,0.25);
+
+  &:before,
+  &:after {
+    content: "";
+    display: block;
+    width: 0;
+    height: 0;
+    border-left: $x-2 solid transparent;
+    border-right: $x-2 solid transparent;
+    border-bottom: $x-2 solid #fff;
+    position: absolute;
+    left: 50%;
+    margin-left: -$x-1;
+  }
+
+  &:before {
+    top: -$x-2;
+    z-index: 2;
+  }
+
+  &:after {
+    border-bottom-color: $ehw-grey-light;
+    top: -($x-2 + 1);
+    z-index: 1;
+  }
+}
+
+.ShareBox .Input__label {
+  border-bottom-color: $ehw-grey;
+  color: $ehw-grey;
+}
+
+.ShareBox__services {
+  clear: left;
+
+  label {
+    position: relative;
+    display: block;
+    width: 100%;
+    padding: 0 0 $x-4;
+    text-transform: uppercase;
+    letter-spacing: 0.16em;
+    font-size: 9px;
+    line-height: 1;
+    color: $ehw-grey-dark;
+  }
+}
+
+.ShareBox__divider {
+  display: block;
+  text-align: center;
+  overflow: hidden;
+  white-space: nowrap;
+  clear: both;
+  text-transform: uppercase;
+  padding: $x-4 0;
+
+  > span {
+    position: relative;
+    display: inline-block;
+  }
+
+  > span:before,
```

Comment:
These could be nested inside above `> span` group by using `&:before, &:after` etc.  Just makes maintenance a little easier in case your 'Or' element ever changes to a div or named element etc.